kafka-ui-react-app/src/components/Topics/shared/Form/TopicForm.tsx

```diff
@@ -52,26 +52,28 @@ const TopicForm: React.FC<Props> = ({
               </p>
             </div>
 
-            <div className="column">
-              <label className="label">Number of partitions *</label>
-              <input
-                className="input"
-                type="number"
-                placeholder="Number of partitions"
-                defaultValue="1"
-                {...register('partitions', {
-                  required: 'Number of partitions is required.',
-                })}
-              />
-              <p className="help is-danger">
-                <ErrorMessage errors={errors} name="partitions" />
-              </p>
-            </div>
+            {!isEditing && (
```

Comment:
We should move fields for _replication factor_ and _partitions count_ to the danger zone only if we edit already existing topic, and not create a new one. Otherwise, these fields should remain where they were before. I implement this logic by passing this boolean flag as a prop